modules/ROOT/pages/mule-oauth-provider-landing-page.adoc

```diff
@@ -12,67 +12,66 @@ endif::[]
-It is the recommended initial solution pending future corporate investments in enterprise specific OAuth 2.0 offerings.
+The Mule OAuth 2.0 Provider is an alternative from MuleSoft for the OAuth 2.0 provider, and can be used in any API Platform organization, including the Federated ones. The Mule OAuth 2.0 provider is the first-step solution for your OAuth 2.0 offerings.
 
-Supported third party OAuth 2.0 providers:
+You must deploy the Mule OAuth 2.0 provider to a runtime with API gateway capabilities, for example, Mule runtime engine (Mule) versions 3.x, 4.2.0, or later, with the corresponding organization credentials. The following third party OAuth 2.0 providers are supported:
 
 * PingFederate
 * OpenAM
 * OAuth 2.0 providers supporting Open ID Connect
 
-The xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 access token enforcement policy] is designed to work with the Mule OAuth 2.0 provider.
+The xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 access token enforcement policy] works with the Mule OAuth 2.0 provider.
 
-== Getting the Mule OAuth 2.0 provider
-To run the Mule OAuth 2.0 Provider you have to deploy the provider to a runtime with API Gateway capabilities (E.g. Mule runtime engine v3.x, Mule runtime engine v4.2.0 or later) with the corresponding organization credentials correctly specified.
+== Downloading the Mule OAuth 2.0 Provider
 
-=== Mule v3.x
-You can download the OAuth Provider from Exchange.
+You can download the Mule OAuth 2.0 provider from Anypoint Exchange:
 
-See the https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-external-oauth2-provider/[External OAuth 2.0 server for Anypoint Platform] asset in Exchange to download it.
-
-=== Mule v4.x
-This feature is available from Mule v4.2.0.
-
-See the https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-oauth2-provider/[Mule OAuth 2.0 server for Anypoint Platform] asset in Exchange to download it.
+* Mule v3.x: See the https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-external-oauth2-provider/[External OAuth 2.0 server for Anypoint Platform].
+* Mule v4.x: See the https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-oauth2-provider/[Mule OAuth 2.0 server for Anypoint Platform].
++
+This feature is availble from Mule v4.2.0.
 
 == OAuth 2.0 dance
 
-The authentication process performed by the Mule OAuth 2.0 provider, API, and client application conforms to RFC 6749. This process is called the xref:oauth-dance-about.adoc[OAuth 2.0 dance].
+The authentication performed by the Mule OAuth 2.0 provider, API, and client application conforms to RFC 6749. This authentication process is known as the xref:oauth-dance-about.adoc[OAuth 2.0 dance]. 
+
+*Nahuel--why are we suddenly jumping to grant types and what is a client for the enforcement policy? There seems to be no connection.*
```

Comment:
There is no mention here of the Client ID enforcement policy. The document states for clarity that someone trying to access an API with the OAuth 2.0 policy applied, will be called from that point on as "Client".

Reply:
ok